include/boost/math/tools/color_maps.hpp

```diff
@@ -0,0 +1,156 @@
4057970371,0.6839159522860683,0.6879568949603344,0.6919652822236589,0.6959450042288188,0.6998941469583252,0.7038136863269572,0.7077036076142078,0.7115648684573126,0.7153974874768007,0.7192003105763611,0.7229763232162184,0.7267244865470727,0.7304437173416366,0.7341377518333918,0.7378037499188499,0.7414454244046753,0.7450587348350122,0.7486483346580772,0.7522104746641185,0.755748680542265,0.7592629017974487,0.7627520603974456,0.766216825373388,0.7696582925955131,0.773077107895631,0.7764713899964052,0.7798428195281752,0.7831920638854646,0.7865182091233281,0.7898228934175395,0.7931051428523151,0.7963656261858302,0.7996034565968984,0.8028201310987728,0.8060159189746301,0.8091884512009491,0.8123413974246995,0.8154719410234268,0.8185825533178849,0.8216705444481089,0.8247373765508464,0.8277822680188923,0.8308060259837033,0.8338091392025193,0.8367899778785102,0.8397489706421578,0.8426854482941508,0.8456008173688659,0.8484934838710892,0.8513631273192183,0.8542102862752687,0.8570334035212495,0.8598348399896373,0.8626110724885258,0.8653643579173034,0.8680923147108822,0.8707954368901418,0.8734743717670276,0.8761266641066531,0.8787535738894046,0.8813526771710705,0.8839261632148947,0.8864707162026111,0.8889878746971145,0.8914771762860599,0.8939362684731502,0.896366404995977,0.8987661885216094,0.9011348456797987,0.9034710462469783,0.9057762624345291,0.9080481396866358,0.9102863527112363,0.9124909968821041,0.9146608551755517,0.9167960662705978,0.9188933135427245,0.9209542277904884,0.9229781644169556,0.9249630587811338,0.9269098852966564,0.928815526839391,0.9306829366142825,0.9325070229302802,0.9342897265833519,0.9360290542859102,0.9377243972740386,0.9393770933973468,0.9409842661949558,0.9425456696997273,0.9440594310662802,0.9455271344382453,0.9469451155697922,0.9483136189409812,0.9496329509923357,0.9509013387619204,0.9521193680507284,0.9532833326102016,0.9543957893212712,0.9554530885899511,0.9564564274893136,0.9574039803720095,0.9582930778369493,0.9591252451658657,0.9598979045875052,0.9606124721276129,0.9612663555645452,0.9618587615061226,0.9623887555724971,0.9628559105126825,0.9632594414845331,0.9635970174364671,0.9638686962955997,0.964073330664618,0.9642088982457305,0.9642762050566095,0.9642722292530789,0.9641978734579372,0.9640500241140576,0.9638285003224122,0.9635333683894992,0.9631614522931872,0.962712538770477,0.9621866932888067,0.9615816256657105,0.960894345402275,0.9601234461397462,0.959270237581007,0.958333448735586,0.9573126236007495,0.9562062303425974,0.9550135580216242,0.9537297610630108,0.9523531724953679,0.9508874276947369,0.9493337129963176,0.9476874510448859,0.9459406787196255,0.944103300221897,0.9421780061800054,0.9401426587007371,0.9380209885848168,0.9358000817994506,0.9334816723713976,0.9310720730839354,0.9285606186650813,0.9259594465149981,0.92327053647064,0.9204816890380406,0.9176163146663037,0.9146765909646813,0.9116700174610778,0.9086068264551928,0.9055122790291467,0.9024220298268155,0.8994080090565665,0.8965597700566809,0.894058310302958},
```

Comment:
Now that I think of it, I doubt we're even coming close to achieving double precision resolution. Would it make sense to replace these character by a 32 bit hexfloat representation?

Reply:
We could. I don't think their are many displays in circulation that have a 32 bit color depth.